src/apps/frontend/pages/tasks/index.tsx

```diff
@@ -1,5 +1,59 @@
-import * as React from 'react';
+import React, { useEffect } from 'react';
+import toast from 'react-hot-toast';
 
-const Tasks: React.FC = () => <div></div>;
+import { HeadingMedium, VerticalStackLayout } from '../../components';
+import { useTaskContext } from '../../contexts';
+import { AsyncError } from '../../types';
+
+import TaskHeader from './task-header';
+import TaskSection from './task-section';
+
+const Tasks: React.FC = () => {
+  const onError = (error: AsyncError) => {
+    toast.error(error.message);
+  };
+
+  const {
+    deleteTask, getTasks, isGetTasksLoading, setTasksList, tasksList,
+  } = useTaskContext();
+
+  const fetchTasks = async () => {
+    try {
+      await getTasks();
+    } catch (error) {
+      onError(error as AsyncError);
+    }
+  };
+
+  useEffect(() => {
+    fetchTasks()
+      .then()
```

Comment:
can we remove this `then`, as it is not needed, if we are not doing anything on resolve.